src/libraries/System.Numerics.Tensors/src/System.Numerics.Tensors.csproj

```diff
@@ -2,14 +2,9 @@
   
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppPrevious);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
-    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' != 'true'">$(TargetFrameworks);net7.0;net6.0</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPackable>true</IsPackable>
     <PackageDescription>Provides support for operating over tensors.</PackageDescription>
-    <!-- Disabling baseline validation since this is a brand new package.
-         Once this package has shipped a stable version, the following line
-         should be removed in order to re-enable validation. -->
-    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
```

Comment:
@ViktorHofer, how do I create an appropriate baseline in order to fix the validation errors?

Reply:
From the build log:

> API breaking changes found. If those are intentional, the APICompat suppression file can be updated by rebuilding with '/p:ApiCompatGenerateSuppressionFile=true'

Reply:
@ViktorHofer, which project?  I've tried building both the ref and the src with that, and it has no affect.

Reply:
The packable project, the source project. This should do it:

`dotnet pack src\libraries\System.Numerics.Tensors\src /p:ApiCompatGenerateSuppressionFile=true`.